### ISSUE TITLE
Typo in folder name "schema+boarview"

### ISSUE THREAD

**Issue by Arciteck** [KiKiHUN1/Mega-Schematics-Downloader#5]
The folder it saves to is incorrectly named, "schema+boarview".

Would it be possible to make it so we can choose our own folder to save schematics to?

https://github.com/KiKiHUN1/Mega-Schematics-Downloader/blob/3fcba9645f741a77aa8edd6bbafe9e9707b9b24c/schema/schema/MainWindow.xaml.cs#L327

Very useful software, thanks.

**Comment by KiKiHUN1** [comment]
Yeah just realized i fked up the folder naming on the cloud drive, and the output folder is based on that. 
But good idea, i'll try to make a workaround for selecting a custom path.

**Comment by KiKiHUN1** [comment]
Added cutom download folder. You have to set it at every app launch. The mega folder renaming wont happening, sorry.
https://github.com/KiKiHUN1/Mega-Schematics-Downloader/commit/563d6ca09f788396133bfc6e09b64b1e07670e49
